newrelic/internal/transform/identifiers.go

```diff
@@ -7,6 +7,8 @@ const (
 	errorCodeAttrKey    = "error.code"
 	errorMessageAttrKey = "error.message"
 
+	serviceNameAttrKey = "service.name"
```

Comment:
Just a comment: It's interesting to me that our telemetry sdk adds the `service.name` attr for spans but _not_ for metrics.

Reply:
Yeah, the `ServiceName` is a first class field in the JSON span structure we send to New Relic, so it became a part of the Span struct in the SDK. The inclusion in the metric is just by knowing it needs to be in the free form attributes. Which makes sense why it is kind of the same way in the SDK. Might be worth refactoring the design in the SDK, but the same could be said of the Metric API I guess. :shrug: